talk.tex

```diff
@@ -10,18 +10,133 @@
 
 \begin{document}
 
+
 \begin{frame}
 \maketitle
 \end{frame}
 
+
 \begin{frame}{Outline}
 \begin{block}{SymEngine}
 \begin{itemize}
-\item 1
-\item 2
-\item 3
+\item Introduction
+\item Why C++
```

Comment:
One thing to highlight is why do you need your CAS fast, this is a broader view and also implies to usage of C++ in SymEngine. `Why fast` maybe too generic a heading, `Why C++` works for now but in the content we should highlight the speed aspects